### PR TITLE
Fixes mobile view

### LINKED ISSUE #1573: Problem in mobile view of overview site.
**Actual Behaviour**

Mobile view is disrupted.
screenshot_2017-12-25-18-15-22-786_com android chrome

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes #1573 
Preview Link - http://agbilotia1998.github.io/Webgen-overview/

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@geekyd please review.

**Comment by agbilotia1998** [comment]
@geekyd it is fixed now please check.

**Comment by agbilotia1998** [comment]
@geekyd can you please run a travis build again?

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "html-minifier": "^3.0.2",
     "jsonapi-serializer": "^3.5.3",
     "jsonfile": "^3.0.0",
+    "mocha": "^4.0.1",
```
Comment (aayusharora) [inline package.json]: Why do we have installed mocha here ? @agbilotia1998

Comment (agbilotia1998) [reply package.json]: I was testing if global installation of mocha is causing problem in travis build or not.

**Comment by agbilotia1998** [comment]
@geekyd done please check.

**Comment by agbilotia1998** [comment]
@geekyd done please review!!

**Comment by mariobehling** [comment]
Thanks!

**Pull request merged**